wemake_python_styleguide/logic/walrus.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+import ast
+import sys
+
+
+def get_assigned_expr(node: ast.AST) -> ast.AST:
+    """
+    Helper function to retrieve assigned value from `NamedExpr`.
+
+    If the node is an actual NamedExpr, the assigned value will be returned.
+    For other node type, the original node is returned.
+
+    This code is only executed on ``python3.8+``,
+    because before ``3.8.0`` release
+    there was no such thing as walrus operator.
+    """
+    if sys.version_info >= (3, 8):
+        if isinstance(node, ast.NamedExpr):  # pragma: no cover
```

Comment:
We use `wemake_python_styleguide.compat.nodes.NamedExpr` for this case.
So, you don't have to use extra `if sys.version_info > ...`

Reply:
By the way, we don't use `# pragma: no cover`
See https://sobolevn.me/2020/02/conditional-coverage

We use `# pragma: py-lt-38`

Reply:
Ok, thx for quick review!